pkg/engine/armvariables.go

```diff
@@ -210,6 +210,12 @@ func getK8sMasterVars(cs *api.ContainerService) (map[string]interface{}, error)
 		cloudInitFiles["auditdRules"] = getBase64EncodedGzippedCustomScript(auditdRules, cs)
 	}
 
+	if kubernetesConfig != nil {
+		if kubernetesConfig.NetworkPlugin == NetworkPluginCilium {
+			cloudInitFiles["systemdBPFMount"] = getBase64EncodedGzippedCustomScript(systemdBPFMount, cs)
```

Comment:
Moved this to the `cloudInitFiles` variable object because it's more idiomatic, and it saves ARM variable overhead.